frontend/libs/studio-components/src/components/StudioPageSpinner/StudioPageSpinner.module.css

```diff
@@ -2,3 +2,8 @@
   display: flex;
   flex-direction: column;
 }
+
+.container {
+  width: 100vw;
+  height: 100vh;
```

Comment:
Adding this CSS to the ```.container``` classname works for me 😄 
```css
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  overflow: hidden;
```

Reply:
What do you think @TomasEng ?

Reply:
I guess this is sufficient since this spinner is used on whole pages only, and with this CSS, it will be centralized in the window.